### PR TITLE
DROOLS-1781: replaces findbugs-maven-plugin

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -199,10 +199,12 @@
     <checkstyle.header.extensions>java</checkstyle.header.extensions>
     <enforcer.failOnDuplicatedClasses>true</enforcer.failOnDuplicatedClasses>
     <!-- Set to "true" on every project that has no violations. -->
-    <findbugs.failOnViolation>false</findbugs.failOnViolation>
+    <spotbugs.failOnViolation>false</spotbugs.failOnViolation>
     <!-- org.eclipse.tycho plugin version -->
     <version.org.eclipse.tycho>1.0.0</version.org.eclipse.tycho>
     <version.org.jboss.tools.tycho-plugins>1.0.0</version.org.jboss.tools.tycho-plugins>
+    <!-- replaces failsafe-maven-plugin -->
```
[the remark this answers is not in the document]
Comment (mareknovotny) [reply pom.xml]: +1

File: pom.xml [hunk 2]
```diff
@@ -1307,8 +1310,7 @@
             <maxRank>6</maxRank>
             <effort>Max</effort>
             <xmlOutput>true</xmlOutput>
-            <failOnError>${findbugs.failOnViolation}</failOnError>
-            <excludeFilterFile>findbugs-excludes.xml</excludeFilterFile>
```
[the remark this answers is not in the document]
Comment (mareknovotny) [reply pom.xml]: yep, this one is moved down to individual modules with the affected NPE error analysis.

Could we filter all by default or keep the individual modules to do that? That is certainly trade off question/answer ;)

File: pom.xml [hunk 3]
```diff
@@ -1307,8 +1310,8 @@
...
+	    <failOnError>${spotbugs.failOnViolation}</failOnError>
+	    <excludeFilterFile>spotbugs-excludes.xml</excludeFilterFile>
```
Comment (mbiarnes) [inline pom.xml]: @jhrcek is this correct - I mean without any path?

**Comment by mbiarnes** [comment]
@jhrcek @mareknovotny done so far

File: pom.xml (hunk 3 above)
Comment (mareknovotny) [reply pom.xml]: so we will have the general filter and these in submodules too?

**Pull request merged**